docs/insomnia/unit-testing.md

```diff
@@ -56,9 +85,11 @@ const body = JSON.parse(response.data);
 expect(body).to.be.an('array');
 ```
 
-### How do I test JSON payload properties?
+### Test JSON Payload Properties
 
-Since unit tests rely on the Chai library for unit testing we can test properties easily once we have converted our response payload to JSON:
+Since unit tests rely on the [Chai library](https://www.chaijs.com/api/bdd/) for unit testing, test properties after converting a response payload to JSON.
```

Comment:
```suggestion
Since unit tests rely on the [Chai library](https://www.chaijs.com/api/bdd/) for assertions, test properties after converting a response payload to JSON.
```